src/ast.rs

```diff
@@ -13,6 +13,8 @@
 use crate::{ops::Op, run::Lab, util::deref};
 use std::{collections::BTreeMap, fmt, str::FromStr};
 
+pub mod transform;
```

Comment:
I think this would better as a new top-level module (ast.rs doesn't use it at all).